apply/apply.go

```diff
@@ -0,0 +1,106 @@
+/* Helper package to merge existing Objects with ones we are trying to create
```

Comment:
Move `apply` under `pkg` package

Reply:
I don't have any folder or go package called pkg... Should I create one at top level folder of this repo, and just cp -r the apply/ folder inside it?

Reply:
Yes. It is not a breaking issue, but the `pkg` folder is a convention in Go for packages.